### PR TITLE
Add example of how to add code to git

### PR DESCRIPTION
**Pull request by estensen**
Add complete example of how to add and commit files.
Solves https://github.com/ad-m/github-push-action/issues/8

It is necessary to configure the email and username of the git user before we can commit:

```
Run git commit -m "Update" -a

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: empty ident name (for <runner@fv-az98.0bd3awjzwfyevouzsa04f54kvf.cx.internal.cloudapp.net>) not allowed
##[error]Process completed with exit code 128.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,10 +14,16 @@ jobs:
...
+    - name: Create local changes
+      run: |
         ...
-    - uses: ad-m/github-push-action@master
+    - name: Add files
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        git commit -m "Add changes" -a
+    - name: Commit changes
```
Comment (tenhobi) [inline README.md]: Push changes

File: README.md [hunk 2]
```diff
@@ -14,10 +14,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
-...
-    - run: |
+    - name: Create local changes
+      run: |
         ...
-    - uses: ad-m/github-push-action@master
+    - name: Add files
```
Comment (tenhobi) [inline README.md]: Commit files

**Comment by estensen** [comment]
Agreed! Thanks for feedback.

**Comment by tenhobi** [comment]
Thx for contributing! 👍

**Pull request merged**